trackintel/similarity/measures.py

```diff
@@ -0,0 +1,215 @@
+"""
+Algorithms e_dtw and e_edr adapted from https://github.com/bguillouet/traj-dist
```

Comment:
I like the idea of having a collection of trajectory distance metrics as it is done in measures.py. In calculate_distance_matrix we tried to be as compatible as possible to [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html). I think here it would also be a good idea to try to implement valid metrics so that we can _piggyback _ on the pairwise_distances implementation of scikit (maybe it is even possible to use their parallelization).
I think this is mostly already done here.